lib/annon_api/plugins/scopes.ex

```diff
@@ -32,20 +34,29 @@ defmodule Annon.Plugins.Scopes do
 
     Conn.put_private(conn, :scopes, scopes)
   end
+  # TODO: refactor this method into a Plug of it's own, designing it after plugins/jwt.ex
```

Comment:
Found a TODO tag in a comment: # TODO: refactor this method into a Plug of it's own, designing it after plugins/jwt.ex